src/lib/components/Ideogram.react.js

```diff
@@ -351,6 +351,17 @@ Ideogram.propTypes = {
      */
     annotationsLayout: PropTypes.string,
 
+    /**
+     * One of "absolute" or "relative". The technique to use in scaling the height of histogram bars. The "absolute" value sets bar height relative to tallest bar in all chromosomes,
```

Comment:
Is it possible to break this line after a reasonable number of characters?
Also, if there are only two possible values, I would write `Either "absolute" or "relative".`

Reply:
This is WIP, I only rearranged the order of some props which clearly belonged together, the next step is to make sure their description make sense and are accurate :)

Reply:
Sorry, I didn't review your pull requests in order! I should have started with #146 :)